### PR TITLE
Bump containers storage to v1.15.8

### PR DESCRIPTION
**Pull request by saschagrunert**
As the title says

### CODE REVIEW

File: vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go [hunk 1]
```diff
@@ -370,7 +370,10 @@ func writeCon(fpath string, val string) error {
 	} else {
 		_, err = out.Write(nil)
 	}
-	return err
+	if err != nil {
+		return errors.Wrapf(err, "failed to set %s on procfs", fpath)
+	}
+	return nil
```
Comment (saschagrunert) [inline vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: This change in selinux-go seems to cause a regression on our side (introduced in https://github.com/opencontainers/selinux/pull/60)

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

File: vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go [hunk 2]
```diff
@@ -289,7 +289,10 @@ func SetFileLabel(fpath string, label string) error {
 	if fpath == "" {
 		return ErrEmptyPath
 	}
-	return lsetxattr(fpath, xattrNameSelinux, []byte(label), 0)
+	if err := lsetxattr(fpath, xattrNameSelinux, []byte(label), 0); err != nil {
+		return errors.Wrapf(err, "failed to set file label on %s", fpath)
+	}
+	return nil
```
Comment (saschagrunert) [inline vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: This change in selinux-go seems to cause a regression on our side (introduced in https://github.com/opencontainers/selinux/pull/60)

Comment (saschagrunert) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: @rhatdan do you have any idea how to fix the SELinux related issue?
```
failed to set file label on /var/lib/containers/storage/overlay/ed4a2284370c886d47ef5a6b0a183a7007b67a354a1f1c29fe478eb8e4399a0a/merged: operation not supported
```

Comment (vrothberg) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: I am also hitting it -> https://github.com/cri-o/cri-o/pull/3205

Comment (rhatdan) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: Looks like we should ignore ENOSUP errors.

Comment (rhatdan) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: Was CRI-O examining the error, and checking for ENOSUP?  It needs to use errors.Cause now.

Comment (rhatdan) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: Where is this being called?

Comment (nalind) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: https://github.com/cri-o/cri-o/blob/master/server/label_linux.go#L11, I think.

Comment (saschagrunert) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: I was thinking the same but was not sure if it is an appropriate solution to skip those errors. What is the reason behind it?

I can spin up a follow-up PR tomorrow if nobody catches up first. :)

Comment (nalind) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: In a number of places, we're already comparing the error we get back to syscall.ENOTSUP.  The change would be to pass those errors to `errors.Cause()` to unwrap them before comparing them.

Comment (rhatdan) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: I believe this is the correct fix.

Comment (nalind) [reply vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go]: Opened #3212.  If you prefer to cherry-pick it into this one and close that one, I'm fine with that.

**Pull request merged**